### PR TITLE
Added PyTorch

### PR DESCRIPTION
**Pull request by sangy987**

### CODE REVIEW

**Review by ritwik12 — CHANGES_REQUESTED** [review]
@sangy987  Filename should be PyTorch.yml, you forgot to put .yml at the end

File: _data/projects/PyTorch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+desc: PyTorch is an open source machine learning library, used for applications such as computer vision and NLP
+site: https://pytorch.org/
+tags:
+- python
+- c++
+- cuda
+- machine learning
+upforgrabs:
+  name: help needed
+  link:https://github.com/pytorch/pytorch
```
Comment (ritwik12) [inline _data/projects/PyTorch:11]: There is no label by the name of `help needed`, please check here https://github.com/pytorch/pytorch/labels
Please add a label which is available or create one. You can use 'good first issue' or 'help wanted' which are mentioned there.

Comment (ritwik12) [inline _data/projects/PyTorch:11]: @sangy987  Change the link to something like https://github.com/pytorch/pytorch/labels/good%20first%20issue as per the label name that you will use.

File: _data/projects/PyTorch [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: PyTorch
+desc: PyTorch is an open source machine learning library, used for applications such as computer vision and NLP
+site: https://pytorch.org/
+tags:
```
Comment (ritwik12) [inline _data/projects/PyTorch:4]: You can add more tags like 
```
neural-network
autograd
gpu
numpy
deep-learning
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/PyTorch (hunk 1 above)
Comment (sangy987) [reply _data/projects/PyTorch:11]: okay thanks

**Comment by shiftkey** [comment]
@ritwik12 @sangy987 it looks like all the requested changes ended up in #1477 instead of this PR

**Comment by sangy987** [comment]
Yes, being a beginner I really do not know how to add the changes it in this PR itself. But, I really appreciate the feedback provided.

**Comment by shiftkey** [comment]
> Yes, being a beginner I really do not know how to add the changes it in this PR itself. But, I really appreciate the feedback provided.

For future reference, here's some tips:

 - if you're making changes in the web editor, each file in the Files tab has a context menu in the top-left where you can make further changes



 - this opens up the editor with your current file, so you can make changes based on the feedback



 - Once you're good to commit the changes, creating a new commit will update the current PR



With that in mind, let's take this over to #1477 and see if we're good to merge it.

**Pull request closed without merging**